Unit/bootstrap.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit;
+
+/**
+ * These constants should be defined in the WP Rocket's bootstrap file:
+ *
+ * - WP_ROCKET_PLUGIN_ROOT
+ * - WP_ROCKET_TESTS_FIXTURES_DIR
+ */
+
+define( 'PHPUNIT_WP_ROCKET_FIXTURES_DIR', dirname( __DIR__ ) . '/Fixtures' );
```

Comment:
We used `WPMEDIA_PHPUNIT` for the generic package, should we follow the same naming convention instead of `PHPUNIT_WP_ROCKET`

Reply:
For this one, we do need to make it more specific as it's related to this add-on package.

Why? Our base `wp-media/phpunit` has a `Fixtures` folder too. If in the future we add more base fixtures in there, then we'll need the ability to access them. In that case, we'd likely use a constant for that package, i.e. `WPMEDIA_PHPUNIT_FIXTURES_DIR`.

In this add-on package, we'd want a way to load fixtures from this specific package.

So given that, do you prefer to prefix constants in this add-on package as:

- `WPMEDIA_PHPUNIT_WP_ROCKET_`
- `PHPUNIT_WP_ROCKET_`

??

Reply:
I agree with what you said. I was thinking about the naming specifically, for example `WP_ROCKET_PHPUNIT` for the parts that are specific to this package